src/ProxyManager/Generator/MethodGenerator.php

```diff
@@ -16,15 +18,74 @@
 class MethodGenerator extends ZendMethodGenerator
 {
     /**
-     * {@inheritDoc}
+     * Similar to fromReflection() but without copying the method body and phpdoc.
+     *
+     * @see \Zend\Code\Generator\MethodGenerator::fromReflection
      */
-    public static function fromReflection(MethodReflection $reflectionMethod) : self
+    public static function fromReflectionWithoutBodyAndDocBlock(MethodReflection $reflectionMethod) : self
     {
-        /* @var $method self */
-        $method = parent::fromReflection($reflectionMethod);
```

Comment:
As mentioned before, the `parent::fromReflection()` call should stay, while the docblock should be removed later on, once the generator instance is available.